src/fetch/orderbook.py

```diff
@@ -53,12 +53,13 @@ def get_order_creations(cls) -> DataFrame:
         """
         Fetches and validates Orderbook Reward DataFrame as concatenation from Prod and Staging DB
         """
-        cow_reward_query = """
+        query = """
         select replace(lower(uid::text), '\\x', '0x') as uid,
                date_part('epoch', creation_timestamp)::integer as creation_time
         from trades
-        inner join orders
+        inner join orders o
           on uid = order_uid
+        where o.fee_amount > 0 
```

Comment:
I discovered that we actually have a field `class` (market, limit, liquidity) on the orders table that we could use here.